neuro-sdk/src/neuro_sdk/_config_factory.py

```diff
@@ -235,12 +235,17 @@ async def login_with_passed_config(
             token = data["token"]
             cluster = data["cluster"]
             url = URL(data["url"])
+            org_name = data.get("org_name")
+            project_name = data.get("project_name")
         except (ValueError, KeyError):
             raise ConfigError(f"Data in passed config is malformed: {config_data}")
         await self.login_with_token(token, url=url, timeout=timeout)
         client = await self.get(timeout=timeout)
 
         await client.config.switch_cluster(cluster)
+        await client.config.switch_org(org_name)
```

Comment:
curious whether this line should also be conditional.

Reply:
yes, switch_org accepts none and sets local config w.o. org